src/pages/webhooks/admin-configuration.md

```diff
@@ -21,11 +21,63 @@ The rows of this grid show configuration settings for all registered hooks, both
 
 ## Create a new hook
 
-Click **Add New Webhook** from the grid page to display the form for creating a new hook. If the plugin for the webhook method and type entered into the form has not been generated for the Commerce instance, a warning to run the `webhooks:generate:module` command will appear upon clicking **Save**.
+Click **Add New Webhook** from the grid page to display the form for creating a new hook.
 
 <InlineAlert variant="warning" slots="text" />
 
-On Cloud instances, due to the read-only file system, the `webhooks:generate:module` command cannot be run. If a plugin-type webhook is added through the Admin, the method name and type should be declared in a `webhooks.xml` file so that the required plugin is generated during the build phase of the deployment process.
+If the plugin for the webhook method and type entered into the form have not been generated for the Commerce instance, a warning to run the `webhooks:generate:module` command will appear upon clicking **Save**. You cannot run the `webhooks:generate:module` command on Cloud instances, due to its read-only file system. If you add a plugin-type webhook through the Admin, declare the method name and type in a `webhooks.xml` file.
```

Comment:
We could consider just adding a mention of deploying after declaring the method name and type in a `webhooks.xml` file since it's during the deployment process that the needed plugin class would get generated

Reply:
We decided to leave this as-is. I added a statement to the events and webhooks `generate:module` commands stating they're for on-premises only,